src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -176,10 +176,13 @@ public static void InstallLocalCertificateFromBridge()
                 }
             }
 
-            InstallCertificateToMyStore(certificateToInstall);
-
-            // We also need to install the root cert if we install a local cert
-            InstallRootCertificateFromBridge();
+            // certificateToInstall could be null in the case the user certification exists
```

Comment:
NIT: I don't quite think this is accurate. If !foundUserCertificate, certificateToInstall will not be null. The reason why these calls are here is because the Install*Certificate calls also take the same lock (s_certificateLock)